walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/Event.java

```diff
@@ -93,4 +94,14 @@ private static boolean equals(Event a, Event b) {
     // interface documentation, equals will check for set-equality across all set implementations.
     return a.title.equals(b.title) && a.when.equals(b.when) && a.attendees.equals(b.attendees);
   }
+
+  /**
+   * Enable sort by the start time.
+   */
+    public static final Comparator<Event> defaultComparator = new Comparator<Event> () {
```

Comment:
I recommend naming the Comparator something more inherently understandable, like SORT_BY_START_TIME (as you have in the comment). "defaultComparator" and "compare" doesn't give too much context

Reply:
The Comparator name is updated!